### PR TITLE
feat(appstate-bar): improve screen reader support

### PR DESCRIPTION
**Pull request by Theeraphat-Sorasetsakul**
## Description
improve screen reader support for appstate-bar

jira: [ELF-1745](https://jira.refinitiv.com/browse/ELF-1745)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/appstate-bar/index.ts [hunk 1]
```diff
@@ -98,7 +105,7 @@ export class AppstateBar extends BasicElement {
       <div part="heading">${this.heading}</div>
       <div part="message"><slot></slot></div>
       <div part="right"><slot name="right"></slot></div>
-      <ef-icon part="close"  @tap="${this.clear}" icon="cross"></ef-icon>
+      <ef-icon role="button" part="close"  @tap="${this.clear}" icon="cross" aria-label="${this.t('CLOSE')}"></ef-icon>
```
Comment (bualoy-napat) [inline packages/elements/src/appstate-bar/index.ts:109]: Please add phrasebook for appstate-bar and don't forget to add the export path in package.json.

File: packages/elements/src/appstate-bar/index.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import {
 import { customElement } from '@refinitiv-ui/core/decorators/custom-element.js';
 import { property } from '@refinitiv-ui/core/decorators/property.js';
 import { VERSION } from '../version.js';
+import { translate, Translate } from '@refinitiv-ui/translate';
```
Comment (bualoy-napat) [inline packages/elements/src/appstate-bar/index.ts:12]:
```suggestion
import { translate, Translate } from '@refinitiv-ui/translate';
import '@refinitiv-ui/phrasebook/locale/en/appstate-bar.js';
```

File: packages/elements/src/appstate-bar/index.ts (hunk 1 above)
Comment (Theeraphat-Sorasetsakul) [reply packages/elements/src/appstate-bar/index.ts:109]: Thank! I will add it

**Comment by Theeraphat-Sorasetsakul** [comment]
Found Narrator focus out to body when navigation between part="message" and close icon. I will convert to draft to find a reason.

**Comment by Theeraphat-Sorasetsakul** [comment]
> Found Narrator focus out to body when navigation between part="message" and close icon. I will convert to draft to find a reason.

The problem happens only when use Narrator+Edge. I think we can skip it.

**Review by wsuwt — APPROVED** [review]

**Pull request merged**